educator/package.yaml

```diff
@@ -107,3 +107,6 @@ executables:
       - disciplina-educator
       - loot-log
       - optparse-applicative
+
+data-files:
+  - database/*.sql
```

Comment:
@kirelagin Is it necessary though? `schema.sql` is completely included in the code in compile time via TH, and `fixtures.sql` is used only for manually setting up an Educator database with some data for manual testing. It is necessary only if you want to initialize Postgres via nix, not via leaving it to node itself.

Reply:
Yeah, I initially misread the issue description. It turns out that there is no need to bundle these, so I suggest we close this PR and someone fixes the docs separately.

Reply:
I already included the config fix in #366